content/user-management copy.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "User Management"
+metaTitle: "sensenet - User Management"
+metaDescription: "sensenet User Management"
+---
+
+User management is a core function of the business, that includes user creation, rights management and assigning roles to the users. Effective user management contains adjusting the access rights of the users who change role, get promoted or leave the company. It is important to close these changes in short time, this way we can increase employee productivity.
```

Comment:
It is important to close these changes in short time, this way we can increase employee productivity.
I don't understand the relation here between short reaction time and productivity. Maybe security is more relevant (or integrity).